src/hw_isolation_record/manager.cpp

```diff
@@ -185,7 +185,28 @@ std::optional<sdbusplus::message::object_path> Manager::createEntry(
                           _bus, entryObjPath, *this, recordId, severity,
                           resolved, associationDeftoHw, entityPath)));
 
-        utils::setEnabledProperty(_bus, isolatedHardware, resolved);
+        // TODO:https://github.com/ibm-openbmc/openpower-hw-isolation/issues/39
+        // During "core" checkstop PLDM will be blocked on the DMA transfer of
+        // the dump data and might not honor enabling resolved property of core
+		// D-Bus object hosted by PLDM. This is a small window where it can
+		// happen.
+		//
+        // For now, ignoring the exception thrown, the property will be enabled
+        // again during refresh.
+		//
+        // This code needs to be removed in P11 when the "Core" D-Bus object
+        // will be hosted by entity manager.
+        try
+        {
+            utils::setEnabledProperty(_bus, isolatedHardware, resolved);
+        }
+        catch (const std::exception& ex)
+        {
+            log<level::ERR>(
```

Comment:
I think we have enough traces in this case from the function caller so, can we avoid it?...just add a comment (you can move your LN194 comment to here)

Reply:
this is not in regular flow and happens very rarely, it would be good to leave the trace there